src/github/webhook.py

```diff
@@ -14,8 +14,8 @@
 # https://docs.github.com/en/developers/webhooks-and-events/webhooks/webhook-events-and-payloads#pull_request
 def _handle_pull_request_webhook(payload: dict) -> HttpResponse:
     pull_request_id = payload["pull_request"]["node_id"]
+    pull_request = graphql_client.get_pull_request(pull_request_id)
```

Comment:
Why are we not logging pull_request here when we do it for issue_comments on line 29?

Reply:
the pull_request info is logged when we call `maybe_automerge_pull_request_and_rerun_stale_checks` but the comment info is never logged in the code paths it takes

Reply:
We're only logging 3 pull request fields. Can we log the entire json body? I suspect we'll want to view it since the granularity of the log you added is useful for clear-cut cases but we may want to know which status checks failed which caused the automerge to fail

Reply:
Added logging of pull_request json to the logic function as opposed to logging in each of the webhooks